test/verify/check-realms

```diff
@@ -95,7 +95,9 @@ class TestRealms(MachineCase):
         if self.machine.image in ["rhel-8-1-distropkg"]:
             self.domain_sel = "#system-info-domain a"
         else:
-            self.domain_sel = "#system-info-domain button"
+            self.domain_sel = "#system_information_domain_button"
+
+        self.machine.upload([ "/home/mvo/work/cockpit/dist/systemd" ], "/usr/share/cockpit" )
```

Comment:
Please remember to take this out for the final commit :-)

Reply:
Done.